sell_only_by_packaging/models/sale_order_line.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+
+from odoo import _, api, models
+from odoo.exceptions import ValidationError
+from odoo.tools import float_compare
+
+
+class SaleOrderLine(models.Model):
+
+    _inherit = "sale.order.line"
+
+    @api.constrains(
+        "product_id", "product_packaging_id", "product_packaging_qty", "product_uom_qty"
+    )
+    def _check_product_packaging_sell_only_by_packaging(self):
+        for line in self:
+            if not line.product_id.sell_only_by_packaging:
```

Comment:
```suggestion
            if not line.product_id.sell_only_by_packaging or not line.product_uom_qty:
```
@QuocDuong1306 IMO there's no point to check lines without any quantity assigned. If we are not able to assign 0 to the `product_uom_qty` we get stuck in many circumstances (specially when the order is already confirmed).

Reply:
@QuocDuong1306 Thank you the change